src/LinkButton/types.ts

```diff
@@ -0,0 +1,12 @@
+import React from "react"
+
+export type Props = {
+    /**
+     * The content of the LinkButton
+     */
+    children: React.ReactNode
+    /**
+     * Event that the LinkButton will receive
+     */
+    onClick?: () => void
+} 
```

Comment:
Desse jeito podemos ter todos os props de um button sem precisar definir um por um.

```suggestion
export type Props = ComponentWithoutRef<"button">
```